head/src/main/kotlin/io/qalipsis/core/head/report/DataSeriesServiceImpl.kt

```diff
@@ -196,4 +203,27 @@ internal class DataSeriesServiceImpl(
         )
     }
 
+    override suspend fun refresh() {
+        orchestrationCoroutineScope.launch {
+            val dataSeriesEntities = dataSeriesRepository.findAll()
```

Comment:
If there are really a lot, you should probably use pagination: https://micronaut-projects.github.io/micronaut-data/latest/guide/#pagination